content/participants/ttjc@DungeonMaster976/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Samruddha Prabhu
+type: "participant"
+name: "Samruddha Prabhu"
+portfolio: "https://"
+github-link: "https://github.com/DungeonMaster976"
+linkedin-link: "https://www.linkedin.com/in/Samruddha-prabhu-78276314a/"
+looking-for: "job / internship"
+description: "JavaScript | React | React Native | Node | GraphQl | "
+---
+
+Hi! I am Samruddha Prabhu Btech 3rd yr from NITK Surathkal let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I am interested in startups. Want to build my own company someday. Learning webdevelopment also native apps. learning MERN stack for teamtanay job challenge. Also have plans to learn ML, AI.
+
+## why are you taking this challenge?
+
+I need an internship or job. I want to learn.Also never tried big projects so this challenge will help me to build good profile.
+
+## something interesting about you?
+
+ I was the same person who asked about building uber like app. Tanay Bhaiya also made a video about it.
```

Comment:
Please add something which is interesting about you here. You can check other participant's entries to get a good idea for it.